### PR TITLE
feat(report): cost center wise financial statements added

### PR DESCRIPTION
**Pull request by vishdha**
TASK ID: https://bloomstack.com/desk#Form/Task/TASK-2020-01908

Report contains cost center wise profit and loss and cost center wise balance sheet

Peek 2020-10-20 13-22

Screenshot:
Accounting

Cost-Center-Wise-Financial-Statements

### CODE REVIEW

File: erpnext/accounts/report/cost_center_wise_profit_and_loss_statement/cost_center_wise_profit_and_loss_statement.py [hunk 1]
```diff
@@ -0,0 +1,64 @@
...
+		"width": 300
+	}]
+
+	columns.append({
+		"fieldname": "currency",
+		"label": _("Currency"),
+		"fieldtype": "Link",
+		"options": "Currency",
+		"hidden": 1
+	})
```
Comment (hrwX) [inline erpnext/accounts/report/cost_center_wise_profit_and_loss_statement/cost_center_wise_profit_and_loss_statement.py]:
```suggestion
	columns = [{
		"fieldname": "account",
		"label": _("Account"),
		"fieldtype": "Link",
		"options": "Account",
		"width": 300
	},
	{
		"fieldname": "currency",
		"label": _("Currency"),
		"fieldtype": "Link",
		"options": "Currency",
		"hidden": 1
	}]
```

**Review by hrwX — CHANGES_REQUESTED** [review]

File: erpnext/config/accounts.py [hunk 2]
```diff
@@ -344,6 +344,12 @@ def get_data():
 					"doctype": "GL Entry",
 					"is_query_report": True
 				},
+				{
+					"type": "report",
+					"name": "Cost Center Wise Financial Statements",
+					"doctype": "GL Entry",
+					"is_query_report": True
+				},
```
Comment (rvpasari) [inline erpnext/config/accounts.py:352]: This looks fine.

File: erpnext/accounts/report/cost_center_wise_financial_statements/cost_center_wise_financial_statements.py [hunk 3]
```diff
@@ -0,0 +1,69 @@
...
+	columns.append({
+		"fieldname": "total",
+		"label": _("Total"),
+		"fieldtype": "Currency",
+		"options": "Currency",
+		"width": 150
+	})
+
+	return columns
+
```
Comment (rvpasari) [inline erpnext/accounts/report/cost_center_wise_financial_statements/cost_center_wise_financial_statements.py:69]: This also looks fines in principle. I guess using it once will help in better determinining issues if any.

File: erpnext/accounts/report/cost_center_wise_financial_statements/cost_center_wise_financial_statements.json [hunk 4]
```diff
@@ -0,0 +1,30 @@
...
+  {
+   "role": "Accounts User"
+  },
+  {
+   "role": "Accounts Manager"
+  },
+  {
+   "role": "Auditor"
+  }
+ ]
```
Comment (rvpasari) [inline erpnext/accounts/report/cost_center_wise_financial_statements/cost_center_wise_financial_statements.json:29]: Should any other users be given access to the document? Administrator for example (if not done by default)

Comment (vishdha) [reply erpnext/accounts/report/cost_center_wise_financial_statements/cost_center_wise_financial_statements.json:29]: @rvpasari  administrator has all role by default.  this is account related report should be access by accounts role

**Review by neilLasrado — APPROVED** [review]

File: erpnext/accounts/report/cost_center_wise_financial_statements/cost_center_wise_financial_statements.py (hunk 3 above)
Comment (neilLasrado) [reply erpnext/accounts/report/cost_center_wise_financial_statements/cost_center_wise_financial_statements.py:69]: True, we will get more feedback once people use this.

**Pull request merged**